slsa/buildtype.go

```diff
@@ -179,6 +181,41 @@ func (b *GithubActionsBuild) Invocation(ctx context.Context) (slsa.ProvenanceInv
 	// workflow run.
 	addEnvKeyString(env, "github_sha1", b.Context.SHA)
 
+	// github_repository_owner is the owner of the repository.
+	addEnvKeyString(env, "github_repository_owner", b.Context.RepositoryOwner)
+
+	oidcClient, err := b.Clients.OIDCClient()
+	if err != nil {
+		return i, fmt.Errorf("oidc client: %w", err)
+	}
+
+	if oidcClient != nil {
+		t, err := oidcClient.Token(ctx, []string{b.Context.Repository})
+		if err != nil {
+			return i, err
+		}
+
+		if t.RepositoryID == "" {
```

Comment:
Are these checks generic enough that they could go in `verifyToken` on the `OIDCClient`?

https://github.com/laurentsimon/slsa-github-generator/blob/22cc114346c587b0751eb61baf01ca52d8cca197/github/oidc.go#L172-L190

Reply:
You're right. Let me move them there. Thanks!

Reply:
done. PTAL

Reply:
Sorry, I mis-spoke. These fields are not generic OIDC fields, they are GitHub-specific claims. I can add them in a `verifyClaims()` function instead?

Reply:
Yeah, either way is fine. this OIDCClient is Github specific so verifying the claims as well would have been ok as well imho but `verifyClaims` works too.